src/main/java/com/zufar/onlinestore/product/endpoint/ProductsEndpoint.java

```diff
@@ -25,13 +28,15 @@
 @Validated
 @RequestMapping(value = "/api/products")
 public class ProductsEndpoint {
+
     private final ProductInfoRepository productInfoRepository;
```

Comment:
delete `productInfoRepository` and `productInfoDtoConverter` from `ProductsEndpoint` class, repository and converter classes should be injected to service class like `ProductInfoService`